irods_specific_implementation/CMakeLists.txt

```diff
@@ -73,6 +72,9 @@ set(CPACK_DEBIAN_PACKAGE_PRIORITY "extra")
 set(CPACK_DEBIAN_PACKAGE_HOMEPAGE "https://irods.org")
 set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS OFF)
 set(CPACK_DEBIAN_PACKAGE_CONTROL_STRICT_PERMISSION ON)
+set(CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA "${CMAKE_SOURCE_DIR}/packaging/postinst;")
+set(CPACK_DEBIAN_PACKAGE_BREAKS ${IRODS_PACKAGE_NAME})
+set(CPACK_DEBIAN_PACKAGE_REPLACES ${IRODS_PACKAGE_NAME})
```

Comment:
don't think we want these two, need some @SwooshyCueb eyeballs

Reply:
Okay. I'll remove these.

Reply:
done